### ISSUE TITLE
There's a undeclared dependency on byebug

### ISSUE THREAD

**Issue by kylegearhart** [story-branch/story_branch#25]

**Comment by rpbaltazar** [comment]
I believe the dependency that you're talking about exists https://github.com/jasonm23/story_branch/blob/master/story_branch.gemspec#L28

**Comment by jasonm23** [comment]
I think this may be referring to the missing Pry dependency, not byebug. ( @rpbaltazar  I was working with @kylegearhart when this was reported)

**Comment by jasonm23** [comment]
```
rspec
bundler: failed to load command: rspec (/usr/local/opt/rbenv/versions/2.4.1/bin/rspec)
LoadError: cannot load such file -- pry
  /Users/jason/workspace/story_branch/spec/spec_helper.rb:2:in `require'
  /Users/jason/workspace/story_branch/spec/spec_helper.rb:2:in `<top (required)>'
  /Users/jason/workspace/story_branch/spec/story_branch_spec.rb:1:in `require'
  /Users/jason/workspace/story_branch/spec/story_branch_spec.rb:1:in `<top (required)>'
  /usr/local/Cellar/rbenv/1.1.0/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.0.4/lib/rspec/core/configuration.rb:1058:in `load'
  /usr/local/Cellar/rbenv/1.1.0/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.0.4/lib/rspec/core/configuration.rb:1058:in `block in load_spec_files'
  /usr/local/Cellar/rbenv/1.1.0/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.0.4/lib/rspec/core/configuration.rb:1058:in `each'
  /usr/local/Cellar/rbenv/1.1.0/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.0.4/lib/rspec/core/configuration.rb:1058:in `load_spec_files'
  /usr/local/Cellar/rbenv/1.1.0/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.0.4/lib/rspec/core/runner.rb:97:in `setup'
  /usr/local/Cellar/rbenv/1.1.0/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.0.4/lib/rspec/core/runner.rb:85:in `run'
  /usr/local/Cellar/rbenv/1.1.0/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.0.4/lib/rspec/core/runner.rb:70:in `run'
  /usr/local/Cellar/rbenv/1.1.0/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.0.4/lib/rspec/core/runner.rb:38:in `invoke'
  /usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.0.4/exe/rspec:4:in `<top (required)>'
  /usr/local/opt/rbenv/versions/2.4.1/bin/rspec:22:in `load'
  /usr/local/opt/rbenv/versions/2.4.1/bin/rspec:22:in `<top (required)>'
```

**Comment by jasonm23** [comment]
Fixed in 8390dfbd5d98177bacefe1e6213c3ce5af7ebc8e
